src/level_gen.cpp

```diff
@@ -225,9 +229,9 @@ static Entity makeStopSign(Engine &ctx, const MapVector2 &p1) {
     float y1 = p1.y;
 
     auto stop_sign = ctx.makeRenderableEntity<PhysicsEntity>();
-    ctx.get<Position>(stop_sign) = Vector3{.x = x1 - ctx.data().mean.x, .y = y1 - ctx.data().mean.y, .z = 0.5};
+    ctx.get<Position>(stop_sign) = Vector3{.x = x1 - ctx.data().mean.x, .y = y1 - ctx.data().mean.y, .z = 1};
```

Comment:
why this value?

Reply:
Stop signs are at the same level as agents.